tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/dynamic-page-validator/dpv-validators/completion-tags-validator.js

```diff
@@ -0,0 +1,49 @@
+import { CleanText } from "../../../helpers/text-helpers";
+import { getSubmissionTimeText } from "../dpv-helpers/time-helpers";
+
+
+export const selfAssessmentPageNewRecommendation = (section, maturity) => {
```

Comment:
This file is called `completion-tags-validator.js` but the function name suggest it's for testing the self-assessment page when a new recommendation happens.

It would be good to change one/both of them to be consistent.

Reply:
done :)